src/silo/storage/sequence_store.cpp

```diff
@@ -104,7 +104,7 @@ void silo::SequenceStorePartition::fillNBitmaps(const std::vector<std::string>&
    tbb::blocked_range<unsigned> const range(0, genomes.size());
    tbb::parallel_for(range, [&](const decltype(range)& local) {
       /// For every symbol, calculate all sequence IDs that have that symbol at that position
-      std::vector<unsigned> positions_with_nucleotide_symbol_n(SYMBOL_COUNT);
```

Comment:
Note the change in the test cases is due to this bugfix: I accidentally always inserted `SYMBOL_COUNT` zeros into the `positions_with_nucleotide_symbol_n`, making the position `0` of all sequences contain the symbol `N`